skorch/tests/callbacks/test_logging.py

```diff
@@ -16,10 +18,12 @@
 from skorch.tests.conftest import tensorboard_installed
 from skorch.tests.conftest import mlflow_installed
 
-
 @pytest.mark.skipif(
     not neptune_installed, reason='neptune is not installed')
 class TestNeptune:
+    # fields logged by on_train_begin and on_train_end
```

Comment:
Regarding the `Error in atexit._run_exitfuncs:` error:

> This is not a problem, they can be ignored. The errors raise because we are using async calls and pytest does not handle closing them well. If I understand correctly, the tests have passed but you just saw this in the logs?

Yes, the tests have passed but the message is confusing. I'm worried that future users (me included) will be confused by this message and think something went wrong. If you know of any way to turn those off, that would be appreciated.

What's strange is that they don't always appear:

```python
pytest -k TestNeptune           # message
pytest -k TestNeptune -s        # no message
pytest -k test_logging          # no message
pytest  # i.e. running all tests, no message

pytest -k "TestTensorboard or TestNeptune"  # message
pytest -k "TestNeptune or TestTensorboard"  # message
pytest -k "TestNeptune or TestSacred"       # message
pytest -k "TestNeptune or TestWandB"        # no message
```

So thankfully it does not appear most of the times, only sometimes if only Neptune or Neptune + a few specific others are tested. This means that a user is most likely to see the message when working with the Neptune tests specifically. If the message cannot be turned off, could you please add a comment here to indicate that the message can be ignored?

Reply:
@BenjaminBossan I think I found the cause for those messages to appear in the tests and fixed it. I don't see them when testing it locally, so please comment if you see it on your side.